cirq-web/cirq_ts/dist/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <title>Cirq Web Development page</title>
+</head>
+<body>
+  <div id="container"></div>
+  <script src="bloch_sphere.bundle.js"></script>
+  <script>
+    CirqTS.blochSphere('{"radius": 5}');
```

Comment:
This shouldn't be CirqTS though, I think this would read better as `BlochSphere.create(...)`

Reply:
CirqTS is the [bundle's library name](https://github.com/seunomonije/Cirq/blob/d59d56aa67cdd2bf7ff922d2c91dc83938bd2937/cirq-web/cirq_ts/webpack.config.js#L30), as we add more files I was thinking that we could have something like `CirqTS.3DCircuit(...)`, `CirqTS.LatticeViz(...)`, etc. But as we add more files to the bundle every function in each visualizations `main.ts` would be accessible by calling `CirqTS.functionName()`.

Reply:
I also noticed the comments at the top of the linked `webpack.config.js` and fixed them locally.

Reply:
Oh, but then the bundle is called  `<script src="bloch_sphere.bundle.js"></script>`. I thought we'll have separate bundles...or all bundles will create the same library name? That sounds like they might conflict with each other?

Reply:
In my head all the bundles would be under the same library name, and we would deal with conflicts like shown [here](https://webpack.js.org/configuration/output/#outputlibrary). If we import only one bundle it shouldn't be a problem, if we import multiple than we can specify. I can also clean up the config to reflect this

Reply:
I actually thought about this more and I actually agree with having no name for the library and just relying on referring to the `bundle.js`...changing this now